.github/workflows/buildAndTest.yml

```diff
@@ -82,6 +82,7 @@ jobs:
             -DCMAKE_CXX_VISIBILITY_PRESET=hidden \
             -DAIE_COMPILER=NONE \
             -DAIE_LINKER=NONE \
+            -DAIE_ENABLE_AIRBIN=ON \
```

Comment:
Because, for example, it's not trivial to get libelf on a MacBook (where myself and others often do dev). Note I'm being explicit in the CI yamls but this option is `ON` by default if the user has libelf (and CMake can find it).

Reply:
ok, got it